apisix/plugins/request-id.lua

```diff
@@ -40,7 +40,9 @@ local schema = {
     properties = {
         header_name = {type = "string", default = "X-Request-Id"},
         include_in_response = {type = "boolean", default = true},
-        algorithm = {type = "string", enum = {"uuid", "snowflake", "nanoid"}, default = "uuid"}
+        algorithm = {type = "string", enum = {"uuid", "snowflake", "nanoid"}, default = "uuid"},
+        id_name = {type = "string", default = "req-id"},
+        get_from_query = {type = "boolean", default = false}
```

Comment:
I think this two fields can be merged into one: `"arg_name"`, which will be better since we already have a `header_name` field, and they're exclusive from each other.

Reply:
I think so